src/nav/ChatNavBar.js

```diff
@@ -56,7 +54,7 @@ export default function ChatNavBar(props: Props) {
         <ExtraButton color={color} narrow={narrow} />
         <InfoButton color={color} narrow={narrow} />
       </View>
-      <LoadingBanner spinnerColor={spinnerColor} backgroundColor={streamColor} textColor={color} />
+      <LoadingBanner unfilledColor={streamColor} color={color} />
```

Comment:
Can you post a gif of what this looks like, with a couple different colors of streams? It seems conceivable that this would look pretty weird.

One thing that's concerning is that `foregroundColorFromBackground` returns `'black' | 'white'`, but I think if this is white, the bar might be invisible?

My suspicion is that just having the bar always be `BRAND_COLOR` (by just not passing in a color) would probably be better, and we can deal with it looking bad when a stream happens to be close to `BRAND_COLOR`, but I'd like to see gifs first. Could you post one gif for each of `black`, `white`, and `BRAND_COLOR`?

Reply:
here they are
image | image |
| --- | --- |
| <img src="https://user-images.githubusercontent.com/31806059/113896032-30638480-97e7-11eb-95df-a434e158ed7a.gif" height="640" /> | <img src="https://user-images.githubusercontent.com/31806059/113896051-35283880-97e7-11eb-86c2-f036b76ecadb.gif " height="640" /> |
|<img src="https://user-images.githubusercontent.com/31806059/113896084-39ecec80-97e7-11eb-9a9e-f4d3d56cf685.gif" height="640" /> | <img src="https://user-images.githubusercontent.com/31806059/113900807-d87b4c80-97eb-11eb-8a22-1399107be956.gif" height="640" /> |

it is same as previously implemented and you already know that  ` <LoadingBanner unfilledColor={streamColor} color={color} />`, | 
accordig to what I have interpreted from review i think loading-bar always have `BRAND_COLOR` would not be a good idea in case of stream's NavBar, and the visuals of loading-bar always same as Navar and their texts, navBar color is the unfilled color of bar and text color will be the animating bar.
As in all other screen it is `BRAND_COLOR` but if we always apply `BRAND_COLOR`  in streams then we have a white horizontal line inside that the bar with `BRAND_COLOR` animate other wise it will be not visible with some colors.

when stream comes near to `BRAND_COLOR` their text also have color that can visible on navbar and the same way this loading-bar will behave.
I just caught, bar will be invisible if `streamColor =undefined` and `color='white'` see image below it would look something like
<img src="https://user-images.githubusercontent.com/31806059/113904908-301bb700-97f0-11eb-8ad9-63e0abb40559.png" />
now we can have a condition  `if streamColor =undefined ` then `color=BRAND_COLOR` then BRAND_COLOR bar animating inside white horizontal view and be visible

Reply:
From discussion on [CZO thread](https://chat.zulip.org/#narrow/stream/48-mobile/topic/loading.20bar.20colors) , come up with these, after adding 1px to height
| | |
|---|---|
| <img src="https://user-images.githubusercontent.com/31806059/114083151-44cd7d00-98cc-11eb-8dde-de020a51ef4f.jpg" /> | <img src="https://user-images.githubusercontent.com/31806059/114083162-472fd700-98cc-11eb-9647-06e102bda768.jpg" /> |
|<img src="https://user-images.githubusercontent.com/31806059/114083170-48f99a80-98cc-11eb-9060-4e4c1a8ae585.jpg" />|<img src="https://user-images.githubusercontent.com/31806059/114083183-4ac35e00-98cc-11eb-8665-3eee25f8bce8.jpg" />|